src/glossary.js

```diff
@@ -89,21 +93,48 @@ function Glossary(terms, selectors) {
   this.addEventListener(document.body, 'keyup', this.handleKeyup.bind(this));
 }
 
-/** Populate internal list.js list of terms */
 Glossary.prototype.populate = function() {
+  var self = this;
+  var terms = this.terms.map(function(term) {
+    return {
+      'term': term.term,
+      'definition': term.definition,
+      'termClass': self.classes.termClass,
+      'definitionClass': self.classes.definitionClass
+    };
+  });
+
+  var itemTemplate = _.template(
+    '<li>' +
+      '<button class="data-glossary-term {{ termClass }}">{{ term }}' +
+      '</button>' +
+      '<div class="{{ definitionClass }}">{{ definition }}</div>' +
+    '</li>',
+    {interpolate: /\{\{(.+?)\}\}/g}
+  );
+
+  terms.forEach(function(term) {
```

Comment:
Looks like we're defining a list of objects called `terms`, and then adding them in a `forEach`. How about just the `forEach`:

```javascript
this.terms.forEach(function(term) {
  var opts = {
    term: term.term,
    definition: term.definition,
    ...
  };
  this.list.insertAdjacentHTML('beforeend', itemTemplate(opts));
});
```